src/main/kotlin/org/aarbizu/baseballDatabankFrontend/records/PaginatedRecords.kt

```diff
@@ -17,29 +19,23 @@ class PaginatedRecords(
     private val pageSize: Int = 10
 ) {
     private val size = records.size
-    private var from: Int = 0
-    private var to: Int = if (size < pageSize) size else from + pageSize
+    private var from = 0
+    private var to = min(from + pageSize, size)
```

Comment:
replace first `pageSize` with `from + pageSize` and then simplify(?) to `min` logic